packages/react/src/side-nav/docs/overview.mdx

```diff
@@ -8,67 +8,71 @@ relatedComponents: ['inpage-nav', 'link-list', 'sub-nav']
 ---
 
 ```jsx live
-<SideNav
-	title="Lodging your tax return"
-	titleLink="#"
-	activePath="#in-detail"
-	items={[
-		{
-			href: '#welcome',
-			label: 'Welcome',
-		},
-		{
-			href: '#do-you-need-to-lodge-a-tax-return',
-			label: 'Do you need to lodge a tax return?',
-		},
-		{
-			href: '#lodge-online',
-			label: 'Lodge online',
-			items: [
+<Columns>
+	<Column columnSpan={{ xs: 12, md: 10, lg: 8, xl: 6 }}>
```

Comment:
Be consistent with our preferred usage?
```suggestion
	<Column columnSpan={{ xs: 12, md: 4, lg: 3 }}>
```

Reply:
Oh, because it's not a full width boooooo. Feels like incorrect advice then…

Reply:
I wonder if changing `<Columns cols={6}>` with the standard `columnSpan` to make it a bit more consistent?

Could also add a comment in the live code?

Reply:
Yeah, that might be a good approach, I'll try that.

- [ ] cols=6